### ISSUE TITLE
Installation method go install does not work as expected

### ISSUE THREAD

**Issue by yigithankarabulut** [vigo/git-init-githubrepo#4]
I think main.go is under cmd/ folder so it doesn't work clear. 

```go install github.com/vigo/git-init-githubrepo/cmd@latest``` I got the binary only when I tried this way but this time the name of the executable was cmd.

a similar issue: https://github.com/golang/go/issues/59281

What is the reason and solution, sir? Tell us.

```bash
ykarabul@ykarabul:~$ go version
go version go1.23.1 linux/amd64

ykarabul@ykarabul:~$ go install github.com/vigo/git-init-githubrepo@latest
go: github.com/vigo/git-init-githubrepo@latest: module github.com/vigo/git-init-githubrepo@latest found (v0.3.4), but does not contain package github.com/vigo/git-init-githubrepo

ykarabul@ykarabul:~$ go install -v github.com/vigo/git-init-githubrepo@latest
go: github.com/vigo/git-init-githubrepo@latest: module github.com/vigo/git-init-githubrepo@latest found (v0.3.4), but does not contain package github.com/vigo/git-init-githubrepo

ykarabul@ykarabul:~$ ls go/bin/
gci      goimports      golines  protoc-gen-go       shadow       swag
gofumpt  golangci-lint  gopls    protoc-gen-go-grpc  staticcheck

ykarabul@ykarabul:~$ go install github.com/vigo/git-init-githubrepo/cmd/git-init-githubrepo@latest
go: github.com/vigo/git-init-githubrepo/cmd/git-init-githubrepo@latest: module github.com/vigo/git-init-githubrepo@latest found (v0.3.4), but does not contain package github.com/vigo/git-init-githubrepo/cmd/git-init-githubrepo

ykarabul@ykarabul:~$ go install github.com/vigo/git-init-githubrepo/cmd@latest

ykarabul@ykarabul:~$ ls go/bin/
cmd  gofumpt    golangci-lint  gopls          protoc-gen-go-grpc  staticcheck
gci  goimports  golines        protoc-gen-go  shadow              swag

ykarabul@ykarabul:~$ cmd -h
NAME:
   cmd - create GitHub friendly git repository with built-in README, LICENSE and more...

USAGE:
   cmd [global options] command [command options] [arguments...]

VERSION:
   0.3.4

AUTHOR:
   Uğur “vigo” Özyılmazel <ugurozyilmazel@gmail.com>

```

**Comment by vigo** [comment]
aaaah... i know the problem, thanks for the heads up, i was using custom brew tap :) ... will fix it asap. thank you @yigithankarabulut

**Comment by yigithankarabulut** [comment]
The problem has been solved with the fix #5 . go install command now works. Issue closed.
